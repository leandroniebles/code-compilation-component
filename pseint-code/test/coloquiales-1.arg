--coloquial_conditions=1