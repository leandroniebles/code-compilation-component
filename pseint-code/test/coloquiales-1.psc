Proceso Coloquiales
	Escribir 5*2 ES ENTERO," ",5*2 NO ES ENTERO;
	Escribir 5*2.5 ES ENTERA," ",5*2.5 NO ES ENTERA;
	Escribir 5*2 ES MENOR A 12," ",5*2 NO ES MENOR A 12;
	Escribir 5*2 ES MENOR QUE 8," ",5*2 NO ES MENOR QUE 8;
	Escribir 5*2 ES MAYOR A 12," ",5*2 NO ES MAYOR A 12;
	Escribir 5*2 ES MAYOR QUE 8," ",5*2 NO ES MAYOR QUE 8;
	Escribir 5*2 ES IGUAL O MAYOR A 10," ",5*2 NO ES IGUAL O MAYOR A 10;
	Escribir 5*2 ES IGUAL O MAYOR QUE 10," ",5*2 NO ES IGUAL O MAYOR QUE 12;
	Escribir 5*2 ES IGUAL O MENOR A 8," ",5*2 NO ES IGUAL O MENOR A 8;
	Escribir 5*2 ES IGUAL O MENOR QUE 10," ",5*2 NO ES IGUAL O MENOR QUE 10;
	Escribir 5*2 ES MAYOR O IGUAL A 12," ",5*2 NO ES MAYOR O IGUAL A 12;
	Escribir 5*2 ES MAYOR O IGUAL QUE 10," ",5*2 NO ES MAYOR O IGUAL QUE 10;
	Escribir 5*2 ES MENOR O IGUAL A 8," ",5*2 NO ES MENOR O IGUAL A 8;
	Escribir 5*2 ES MENOR O IGUAL QUE 12," ",5*2 NO ES MENOR O IGUAL QUE 12;
	Escribir 5*2 ES IGUAL A 10," ",5*2 NO ES IGUAL A 10;
	Escribir 5*2 ES IGUAL QUE 8," ",5*2 NO ES IGUAL QUE 8;
	Escribir 5*2 ES DISTINTO A 10," ",5*2 NO ES DISTINTO A 10;
	Escribir 5*2 ES DISTINTO DE 8," ",5*2 NO ES DISTINTO DE 8;
	Escribir 5*2 ES DISTINTA A 10," ",5*2 NO ES DISTINTA A 10;
	Escribir 5*2 ES DISTINTA DE 8," ",5*2 NO ES DISTINTA DE 8;
	Escribir 5*2 ES PAR," ",5*2 NO ES PAR;
	Escribir 5*2 ES IMPAR," ",5*2 NO ES IMPAR;
	Escribir 5*2 ES POSITIVO," ",5*2 NO ES POSITIVO;
	Escribir -5*2 ES POSITIVA," ",-5*2 NO ES POSITIVA;
	Escribir 5*2 ES NEGATIVO," ",5*2 NO ES NEGATIVO;
	Escribir -5*2 ES NEGATIVA," ",-5*2 NO ES NEGATIVA;
	Escribir 5*2 ES CERO," ",5*2 NO ES CERO;
	Escribir 5*2 ES DIVISIBLE POR 2," ",5*2 NO ES DIVISIBLE POR 2;
	Escribir 5*2 ES MULTIPLO DE 3," ",5*2 NO ES MULTIPLO DE 3;
	Escribir 5*2 ES 10," ",5*2 NO ES 10;
FinProceso
