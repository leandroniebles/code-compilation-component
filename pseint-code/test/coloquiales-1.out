VERDADERO FALSO
FALSO VERDADERO
VERDADERO FALSO
FALSO VERDADERO
FALSO VERDADERO
VERDADERO FALSO
VERDADERO FALSO
VERDADERO VERDADERO
FALSO VERDADERO
VERDADERO FALSO
FALSO VERDADERO
VERDADERO FALSO
FALSO VERDADERO
VERDADERO FALSO
VERDADERO FALSO
FALSO VERDADERO
FALSO VERDADERO
VERDADERO FALSO
FALSO VERDADERO
VERDADERO FALSO
VERDADERO FALSO
FALSO VERDADERO
VERDADERO FALSO
FALSO VERDADERO
FALSO VERDADERO
VERDADERO FALSO
FALSO VERDADERO
VERDADERO FALSO
FALSO VERDADERO
VERDADERO FALSO
